pkg/kubeapiserver/admission/config.go

```diff
@@ -17,35 +17,20 @@ limitations under the License.
 package admission
 
 import (
-	"net/http"
 	"os"
 
 	"k8s.io/klog/v2"
 
-	"go.opentelemetry.io/otel/trace"
-
 	"k8s.io/apiserver/pkg/admission"
-	webhookinit "k8s.io/apiserver/pkg/admission/plugin/webhook/initializer"
-	"k8s.io/apiserver/pkg/server/egressselector"
-	"k8s.io/apiserver/pkg/util/webhook"
-	externalinformers "k8s.io/client-go/informers"
-	"k8s.io/client-go/rest"
-	"k8s.io/kubernetes/pkg/kubeapiserver/admission/exclusion"
-	quotainstall "k8s.io/kubernetes/pkg/quota/v1/install"
 )
 
 // Config holds the configuration needed to for initialize the admission plugins
 type Config struct {
-	CloudConfigFile      string
-	LoopbackClientConfig *rest.Config
-	ExternalInformers    externalinformers.SharedInformerFactory
+	CloudConfigFile string
 }
 
 // New sets up the plugins and admission start hooks needed for admission
-func (c *Config) New(proxyTransport *http.Transport, egressSelector *egressselector.EgressSelector, serviceResolver webhook.ServiceResolver, tp trace.TracerProvider) ([]admission.PluginInitializer, error) {
```

Comment:
Note for the review: all these removed lines went into `pkg/controlplane/apiserver/admission`